csrc/scheduler/normalization_utils.cpp

```diff
@@ -1476,24 +1477,47 @@ void schedulePersistentKernel(
     fusion->addOutput(output);
   }
 
-  const bool unroll = rparams->isUnrolled();
-  const bool vectorize =
+  const bool is_unroll_or_vectorization = rparams->isUnrolled();
+  const bool is_vectorize =
       rparams->vectorize_inner_reduction || rparams->vectorize_iter_dom;
-  const bool is_outer_grid_persistence = rparams->persistent_kernel &&
+  const bool use_grouped_reduction = rparams->persistent_kernel &&
```

Comment:
nitpick, why is the naming different here?

we have `is_unroll_...` / `is_vectorize` and this changes it to `use_...`

Reply:
The parameter `use_grouped_reduction` more clearly indicates that grouped reduction is either unavailable or not desired. In contrast, the original name, `is_outer_grid_persistence`, raises questions about why outer grid persistence is being checked. With the new name, it becomes clear that the focus is on handling grouped reduction.